docs/src/tutorials/huggingface.rst

```diff
@@ -0,0 +1,183 @@
+*****************************************************************************************
+Hyperparameters optimisation using a HuggingFace Model and the Hydra-Orion-Sweeper plugin
+*****************************************************************************************
+
+In this tutorial, we will show an easy Orion integration of a HuggingFace translation model using
+Hydra, with the `Hydra_Orion_Sweeper <https://github.com/Epistimio/hydra_orion_sweeper>`_ plugin.
+Hydra is essentially a framework for configuring applications. We will use it to define our
+Hyperparameters and some Orion configuration. We will also be using
+`Comet <https://www.comet.com/>`_ for monitoring our experiments.
+
+Imports
+^^^^^^^
+You will need to import some modules using pip, such as ``datasets``, ``transformers``,
+``hydra-orion-sweeper`` and ``comet-ml``.
+
+.. literalinclude:: /../../examples/huggingface/main.py
+  :language: python
+  :lines: 5-6,13-22
+
+
+Hydra configuration file
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Notice here how the arguments that are not defined will be set at ``None``, which will be
+overridden by default values or not used at all. This serves as a replacement for parsing arguments
+in the command line, but is integrated with Orion, which makes the usability really good.
+
+.. literalinclude:: /../../examples/huggingface/config.yaml
+  :language: yaml
+  :lines: 1-
+
+If you want to change your working or/and logging directory, you can also easily do that.
+From the config file, you can specify
+
+.. literalinclude:: /../../examples/huggingface/config.yaml
+  :language: yaml
+  :lines: 29-31
+
+This will change your working directory. You can see that with the hydra-orion-sweeper, we are able
+to specify 4 variables from Orion. ``${hydra.sweeper.orion.name}`` for the ``experiment_name``,
+``${hydra.sweeper.orion.id}`` for the ``experiment_id``, ``${hydra.sweeper.orion.uuid}`` for the
+experiment ``uuid`` and ``${hydra.sweeper.orion.trial}`` for the ``trial_id``.
```

Comment:
Maybe there should be another sentence explaining that this will create different folders for each trial, under different folders for each exp, and different for each sweep.

Reply:
added in [4a5965d](https://github.com/Epistimio/orion/pull/1040/commits/4a5965dc093261b1747a1a06d75cd67970f9262a)